cogeo_mosaic/scripts/cli.py

```diff
@@ -150,31 +150,40 @@ def create_from_features(
 @cogeo_cli.command(short_help="Create mosaic definition from list of files")
 @click.argument("input_files", type=click.File(mode="r"), default="-")
 @click.argument("input_mosaic", type=click.Path())
-@click.option("--output", "-o", type=click.Path(exists=False), help="Output file name")
 @click.option("--min-tile-cover", type=float, help="Minimum % overlap")
+@click.option(
+    "--on-top/--on-bottom",
+    help="happens dataset on top of the existing scenes.",
+    is_flag=True,
+    default=True,
+)
+@click.option("--overwrite", help="overwrite mosaic.", is_flag=True)
 @click.option(
     "--threads",
     type=int,
     default=lambda: os.environ.get("MAX_THREADS", multiprocessing.cpu_count() * 5),
     help="threads",
 )
-def update(input_files, input_mosaic, output, min_tile_cover, threads):
+def update(input_files, input_mosaic, min_tile_cover, on_top, overwrite, threads):
```

Comment:
So `min_tile_cover` only applies to new assets?

Reply:
yes